vonk/features/customsearchparameters.rst

```diff
@@ -77,21 +82,23 @@ To re-index all resources for certain search parameters, use:
 	::
 	
 		POST http(s)://<vonk-endpoint>administration/reindex/searchparameters
+		Accept=application/fhir+json (or xml); fhirVersion=3.0 (or 4.0)
 
-	In the body of the POST, you put the parameters to actually re-index as form parameters:
+In the body of the POST, you put the parameters to actually re-index as form parameters:
 
 	::
 	
 		include=Patient.name,Observation.code
 		exclude=Organization.name
 
-	``include`` means that resources will be re-indexed only for those search parameters.
-	You use this if you added or changed one or few search parameters.
+``include`` means that resources will be re-indexed only for those search parameters.
+You use this if you added or changed one or few search parameters.
+
+``exclude`` means that any existing index data for those search parameters will be erased.
```

Comment:
For another time: to me it seems strange that we call this parameter 'exclude' instead of 'drop' or 'remove'. Exclude implies that it is just not reindexed, not that it is removed

Reply:
But it also makes sense: the SearchParameter itself is not removed from Administration. You have to that afterwards.